justfile

```diff
@@ -26,6 +26,15 @@ oak_restricted_kernel_bin:
 _wrap_kernel kernel_bin_prefix:
     env --chdir=oak_restricted_kernel_wrapper OAK_RESTRICTED_KERNEL_FILE_NAME={{kernel_bin_prefix}}_bin cargo build --release
     rust-objcopy --output-target=binary oak_restricted_kernel_wrapper/target/x86_64-unknown-none/release/oak_restricted_kernel_wrapper oak_restricted_kernel_wrapper/target/x86_64-unknown-none/release/{{kernel_bin_prefix}}_wrapper_bin
+    rm -rf oak_restricted_kernel_wrapper/target/released_bin_with_components_{{kernel_bin_prefix}}
```

Comment:
I think this logic would benefit from some comment -- I imagine a new starter would find it quite difficult to figure out what's going on and why. We may even want to have a paragraph in some readme and refer to that, since it probably won't fit in an inline comment here.